app/views/items/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="item-form-container">
+  <%= form_for(@item, url: category_items_path(@category), method: :post, html: { class: "item-form" }) do |f| %>
+    <div class="input-field">
+      <%= f.text_field :name,  placeholder: "Enter name" %>
+    </div>
+
+    <br>
+    
+    <div class="input-field">
+      <%= f.number_field :amount, placeholder: "Enter amount" %>
+    </div>
```

Comment:
- [ Required ] I think it would be better if you made the `name` and `amount` input fields required so that a user is not able to add an entry with a blank `name` and `amount`. I actually succeeded in doing this when I was testing the App, this could be improved by just adding the `required` attribute to these fields as shown below.

## Example code
```ruby
<div class="item-form-container">
  <%= form_for(@item, url: category_items_path(@category), method: :post, html: { class: "item-form" }) do |f| %>
    <div class="input-field">
      <%= f.text_field :name,  placeholder: "Enter name", required: true %>
    </div>

    <br>
    
    <div class="input-field">
      <%= f.number_field :amount, placeholder: "Enter amount", required: true %>
    </div>

  <div class="add-btn">
    <%= f.submit "Create Item" %>
  </div>
  <% end %>
</div>
```

## Screenshot
![Screenshot from 2023-08-09 02-18-06](https://github.com/Naledi-Dikgale/ror-budgetapp/assets/16392046/4d9609e7-e4e3-4cf2-a57d-2d54be0deb0c)